### DISCUSSION TITLE
没有找到service的日志

### DISCUSSION THREAD

**Discussion by yujunting** [xmake-io/xmake#5637] [Q&A]
![image](https://github.com/user-attachments/assets/9b2b11f3-61f2-401c-808c-7db4229c7082)
如图

**Comment by waruqi** [comment]
这个只有在作为 `xmake service --daemon` 后台进程时候，才会去写 log

如果没有，你就调下这里 

https://github.com/xmake-io/xmake/blob/b9385560675a37623b2d4d385b6290b87b824d66/xmake/modules/private/service/start_service.lua#L55
